src-srv/sql-gen.js

```diff
@@ -318,16 +318,16 @@ let getSqlFromPatchInternal = function(patch, path, result, accountId) {
       } else if (op === 'ArrayAdd') {
         // We need to add the key back to the object
         let insertObject = {};
-        insertObject[applicableTableReference] = JSON.parse(value.param1);
+        insertObject[applicableTable] = JSON.parse(value.param1);
 
         // Get the parent ids from the path
         let parents = {};
-        if (applicableTableReference == 'games') {
+        if (applicableTable === 'games') {
           parents.teamId = path[1];
-        } else if (applicableTableReference == 'plateAppearances') {
+        } else if (applicableTable === 'plateAppearances') {
           parents.teamId = path[1];
           parents.gameId = path[3];
-        } else if (applicableTableReference == 'lineup') {
+        } else if (applicableTable === 'lineup') {
```

Comment:
Oops. I broke this. I'll merge this PR and fix it properly. The variable applicableTableReference should be the javaValue for the db table the the name of the db table itself.